src/test/kotlin/de/jodamob/kotlin/testrunner/sample/TestWithRunner.kt

```diff
@@ -1,15 +1,17 @@
 package de.jodamob.kotlin.testrunner.sample
 
 import de.jodamob.kotlin.testrunner.KotlinTestRunner
+import de.jodamob.kotlin.testrunner.OpenedClasses
+import de.jodamob.kotlin.testrunner.nonopen.FinalNonOpenClassSample
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.mockito.Mockito
 import org.mockito.exceptions.base.MockitoException
 
 @RunWith(KotlinTestRunner::class)
+@OpenedClasses("de.jodamob.kotlin.testrunner.sample")
```

Comment:
also see 2 use cases:
external depdendencies as you say and then kotlin classes that dont need to be opened so for me this would be everything in may application package even if it slows down performance a bit